spec/features/site/user_visits_home_page_spec.rb

```diff
@@ -3,7 +3,7 @@
 feature 'User visits home page' do
   context 'when they are signed in' do
     background { sign_in_with(user.email, user.password) }
-    given(:user) { create(:user) }
+    given(:user) { create(:user, :moderator) }
```

Comment:
I was going to ask you about this.  Currently I don't have anything in place to check roles.  I saw in some of your apps you use pundit, would you recommend that?  Also should I create a separate issue for locking pages to roles or just add it to this one?